owncloudApp/src/main/res/values/strings.xml

```diff
@@ -726,10 +726,18 @@
     <string name="release_notes_proceed">Proceed</string>
     <string name="release_notes_icon">Release note icon</string>
 
-    <string name="release_notes_4_1_1_title_1">Thumbnails correctly shown for every user</string>
-    <string name="release_notes_4_1_1_subtitle_1">Users with an @ in their username will now be able to see thumbnails correctly</string>
-    <string name="release_notes_4_1_1_title_2">Minor bugfixes</string>
-    <string name="release_notes_4_1_1_subtitle_2">Some bugs were fixed to improve experience in the app</string>
+    <string name="release_notes_4_2_0_title_1">"Apply to all" when many name conflicts arise</string>
+    <string name="release_notes_4_2_0_subtitle_1">A new dialog has been created where a checkbox has been added to be able to select all the folders or files that have conflicts</string>
+    <string name="release_notes_4_2_0_title_2">"Share to" in oCIS accounts allows upload to any space</string>
+    <string name="release_notes_4_2_0_subtitle_2">With this improvement, shared stuff from other apps can be uploaded to any space and not only the personal one in oCIS accounts</string>
+    <string name="release_notes_4_2_0_title_3">Deep link open app correctly</string>
+    <string name="release_notes_4_2_0_subtitle_3">Opening the app with the deep link correctly and managing if user logged or not.</string>
```

Comment:
`A link that points to the app will open the app and then, resolve the file to preview it directly`

Reply:
Done